old_api/__init__.py

```diff
@@ -0,0 +1,66 @@
+import hashlib
+from datetime import datetime, timezone
+
+import requests
+from django.conf import settings
+
+from old_api.schema import FileList, FileMetadata
+
+
+session = requests.Session()
+session.headers["Authorization"] = settings.AIRLOCK_API_TOKEN
+
+
+def create_filelist(request):
+    files = []
+    root = request.root()
+
+    for relpath in request.filelist():
+        abspath = root / relpath
+        stat = abspath.stat()
+        mtime = datetime.fromtimestamp(stat.st_mtime, tz=timezone.utc).isoformat()
+        files.append(
+            FileMetadata(
+                name=str(relpath),
+                size=stat.st_size,
+                sha256=hashlib.sha256(abspath.read_bytes()).hexdigest(),
+                date=mtime,
+                url=str(relpath),  # not needed, but has to be set
+                metadata={},  # not needed *yet*, but can't be None
```

Comment:
`osrelease` sets `tool` in the metadata for the file as well, and that's what my dashboard SQL currently relies on. I think it would be quite reasonable to omit it, but I should update the SQL to join on the Release object if so.